matplotcheck/base.py

```diff
@@ -1157,40 +1157,116 @@ def assert_lines_of_type(self, line_types):
 
     ## HISTOGRAM FUCNTIONS ##
 
+    def get_num_bins(self):
+        """Gets the number of bins in histogram with a unique x-position.
+
+        Returns
+        -------
+        Int :
+            Returns the number of bins with a unique x-position. For a normal
+            histogram, this is just the number of bins. If there are two
+            overlapping or stacked histograms in the same `matplotlib.axis.Axis`
+            object, and the two histograms have the same bin edges, then this
+            function returns the number of bins from one """
```

Comment:
Let's see. returns the number of bins from one. ... can you clarify please?

I think maybe soemthing like
`returns the number of bins from the first set of histogram bins` or soemthing like that?